app/views/contributors/new.html.haml

```diff
@@ -0,0 +1,20 @@
+%h1 Thank you!
+%p
+  This data you contributed will be reviewd
+  by an administrator, and go live shortly.
```

Comment:
I think we can improve this text a bit but that's not a problem for this first pass being merged. We should open an issue for that.